### PR TITLE
Added tests for unrealized P/L

### LINKED ISSUE #7664: Automate Unrealized P/L Test
We've automated realized P/L tests but not any unrealized ones.

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-sdk/src/state/getter/Users.ts [hunk 1]
```diff
@@ -1063,6 +1063,7 @@ export class Users {
       params.account!,
       profitLossOrders
     );
+    console.log('profitLossByMarketAndOutcome', JSON.stringify(profitLossByMarketAndOutcome, null, 2))
```
Comment (justinbarry) [inline packages/augur-sdk/src/state/getter/Users.ts]: Debug code?

Comment (robert-e-davidson3) [reply packages/augur-sdk/src/state/getter/Users.ts]: gah

File: packages/augur-artifacts/src/environments/kovan.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
       "ws": "wss://v2.augur.net/0x-ws"
     },
     "mesh": {
-      "verbosity": 1,
+      "verbosity": 5,
```
Comment (justinbarry) [inline packages/augur-artifacts/src/environments/kovan.json]:
```suggestion
      "verbosity": 1,
```

Do we want it this noisy by default?

Comment (robert-e-davidson3) [reply packages/augur-artifacts/src/environments/kovan.json]: ugh I changed that for testing. one sec...

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**